src/components/CompiledOutput.js

```diff
@@ -226,30 +215,53 @@ export function CompiledOutput({
                           }}
                         />
                       ))}
+                      <Dropdown.Item
+                        text="Output"
+                        onClick={() => {
+                          setTimeTravelCode(sourceCode);
+                          setDisplayAtIndex("Output");
+
+                          // Source output is the first element
+                          if (timeTravelIndex !== timeTravel.length) {
+                            setTimeTravelIndex(1);
+                          }
+                        }}
+                      />
                     </Dropdown.Menu>
                   </Dropdown>
                 ) : null}
               </Menu.Item>
-              <Button
-                content="Next"
-                onClick={() => {
-                  /*
-                  To get the original indices of the array
-                  we reverse the operation earlier.
-                */
-                  setDisplayAtIndex(
-                    `${timeTravel[timeTravelIndex - 1]?.currentNode}`
-                  );
-                  setTimeTravelCode(`${timeTravel[timeTravelIndex - 1]?.code}`);
-                  if (timeTravelIndex !== timeTravel.length) {
-                    setTimeTravelIndex(timeTravelIndex + 1);
-                  }
-                }}
-              />
+              <Menu.Item>
+                <Button
+                  content="Next"
+                  onClick={() => {
+                    /*
+                    To get the original indices of the array
+                    we reverse the operation earlier.
+                  */
+                    setDisplayAtIndex(
+                      `${timeTravel[timeTravelIndex - 1]?.currentNode}`
+                    );
+                    setTimeTravelCode(`${timeTravel[timeTravelIndex - 1]?.code}`);
+                    if (timeTravelIndex !== timeTravel.length) {
+                      setTimeTravelIndex(timeTravelIndex + 1);
+                    }
+                  }}
+                />              </Menu.Item>
```

Comment:
Just for format consistency, can `</Menu.Item>` be moved to the next line

Reply:
I've created a monster. I should have taught you to just blindly approve PRs.